test/epicenter_web/controllers/user_auth_test.exs

```diff
@@ -6,7 +6,7 @@ defmodule EpicenterWeb.UserAuthTest do
   alias Epicenter.AccountsFixtures
   alias Epicenter.Repo
   alias EpicenterWeb.UserAuth
-  alias EpicenterWeb.Router.Helpers, as: Routes
+  use Phoenix.VerifiedRoutes, endpoint: EpicenterWeb.Endpoint, router: EpicenterWeb.Router
```

Comment:
I don't think you need this line because you added it to [EpicenterWeb.ConnCase](https://github.com/RatioPBC/epi-viewpoint/pull/26/files#diff-f267dd4fbe7158f786006d9ae8d70bb75c5fc913a24890f16853c4b7d6bb73a9R34).

Reply:
thanks for pointing that out!